assets/stylesheets/modules/llms/common/ai-llms-editor.scss

```diff
@@ -110,8 +77,8 @@
     grid-template-columns: repeat(auto-fill, minmax(16em, 1fr));
     gap: 1em 2em;
     margin-top: 1em;
-    border-top: 3px solid var(--primary-low); // matches tbody border
```

Comment:
I had a super weird behavior in my editor with this line not sure why, removed it for now